src/index.ts

```diff
@@ -3,7 +3,148 @@
 'use-strict';
 
 import {
+  ICell, ICodeCellViewModel, IMarkdownCellViewModel, IRawCellViewModel
+} from 'jupyter-js-cell';
```

Comment:
I am still not excited about the `-js-` in out pakage/repo names. Will bring this up on the list.